src/app/public/modules/repeater/repeater.module.ts

```diff
@@ -68,6 +73,8 @@ import {
     SkyRepeaterItemTitleComponent
   ],
   providers: [
+    SkyCoreAdapterService,
+    SkyRepeaterAdapterService,
```

Comment:
Isn't this being provided by the repeater component?

Reply:
Sure is. Good catch. I also noticed the same issue for `SkyRepeaterService`, so I removed it.